_includes/layouts/post.html

```diff
@@ -6,17 +6,22 @@
 This is used in blog posts. The index page can be found at blog/index.html
 {% endcomment %}
 
-<div class="usa-layout-docs usa-section">
+<div class="usa-layout-docs usa-section post-layout">
   <div class="grid-container">
     <div class="grid-row grid-gap">
       <div class="usa-layout-docs__main desktop:grid-col-9 usa-prose">
-        <h1 class="title">{{title}}</h1>
+        <h2 class="title">{{title}}</h2>
         <div class="text-base margin-bottom-2">
           <div class="margin-top-neg-105">
-            By <span class="text-bold">{{ author }}</span> · {{ date | date: '%B %d, %Y' }}
+            <!--By <span class="text-bold">{{ author }}</span> · -->{{ date | readableDate }}
           </div>
         </div>
         {{ content }}
+
+        <div class="grid-row">
+          <div class="grid-col flex-4"></div>
+          <div class="grid-col flex-1"><p><a href="{{ site.github_url }}/edit/{{ site.github_branch }}/{{ page.relative_path }}">Suggest edits</a></p></div>
```

Comment:
See .eleventy.js

Reply:
I think you already saw it by now because of another comment, but here: https://github.com/cloud-gov/site/pull/20/files#diff-c306e0a99961a16f5c5c83996caa0958b94006d97f97475049ea3a08036bb5b0

Eleventy now handles external links on all content and templates during the build through a custom function on the markdown-it plugin.

Reply:
> Eleventy now handles external links on all content and templates during the build through a custom function on the markdown-it plugin.

Hmm... That would work on markdown file content parsed by the markdown engine. This part is in a template. Are you sure it still works that way?

Reply:
100%.  Side note, adding the class "usa-link--external" makes footer links that are external very hard to read.  Override it for the footer?  Run local to view or shoot me a message in slack and I'll send you a screenshot.

Reply:
ok so we still need to target _blank stuff here, hardcoded, yes?

Reply:
I got that too.